### DISCUSSION TITLE
subcommands

### DISCUSSION THREAD

**Discussion by giscus** [brainboxdotcc/dpp-comments#20] [General]
# subcommands

A lightweight C++ Discord API library supporting the entire Discord API, including Slash Commands, Voice/Audio, Sharding, Clustering and more!

https://dpp.dev/subcommands.html

**Comment by Stiletti** [comment]
subcommand.get_option<dpp::snowflake>(0)

->** class dpp::command_data:option has no member get_option**

**Reply by braindigitalis** [reply]
> subcommand.get_optiondpp::snowflake(0)
> 
> ->** class dpp::command_data:option has no member get_option**

the comments feature is not for reporting issues, if you have an issue with your code please come to discord for assistance.
https://discord.gg/dpp

**Comment by CZmisaCZ** [comment]
This example does not works

> dpp::command_data:option has no member get_option

**Comment by Ilikepie928** [comment]
is it deprecated because when i pasted the code to see what it does it throws a bunch of errors

**Reply by braindigitalis** [reply]
its get_value not get_option, looks like the example has a typo.

**Reply by braindigitalis** [reply]
https://dpp.dev/structdpp_1_1command__interaction.html#a23a82b95593d4f06b8819773c65454d6

**Reply by Ilikepie928** [reply]
ty

**Comment by Random1O11** [comment]
how do i do this but with a string input instead of a user

**Reply by leeendl** [reply]
replace `dpp::co_user` -> `dpp::co_string`
